integration/stability/bind_mount_linux.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will start a container with a bind mount
+# and set bind propagation, the purpose of this
+# test is to check if the container stops properly
+
+set -e
+
+cidir=$(dirname "$0")
+testname="${0##*/}"
+sysname=$(uname)
+
+[ "$sysname"x == "Linux"x ] || exit 0
+
+source "${cidir}/../../metrics/lib/common.bash"
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+MOUNT_DIR="/mnt/tmp"
+BIND_DIR="${MOUNT_DIR}/sub"
+DOCKER_ARGS="-v ${MOUNT_DIR}:${MOUNT_DIR}:rslave"
+CONTAINER_ID=
+
+# Set the runtime if not set already
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+function setup {
+	clean_env
+	docker run --runtime=$RUNTIME -d $DOCKER_ARGS --name $CONTAINER_NAME $IMAGE $PAYLOAD_ARGS
```

Comment:
can you please quote all variables here and below?

Reply:
update